vars/deployContainerToEnv.groovy

```diff
@@ -1,3 +1,5 @@
+#!/bin/env groovy
+
 def call(Map params) {
     withCredentials([sshUserPrivateKey(credentialsId: '71d94074-215d-4798-8430-40b98e223d8c', keyFileVariable: 'keyFileVariable', passphraseVariable: '', usernameVariable: 'usernameVariable')]) {
```

Comment:
might be worth creating a ticket for the backlog to somehow paramartarize this id